### PR TITLE
Enable tpu device ids test

### PR DESCRIPTION
**Pull request by kaushikb11**
## What does this PR do?

Enable tpu device ids test which was disabled due to timeout issue. (We increased the timeout period to 20 mins)

## Before submitting

- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [x] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @carmocca @akihironitta @borda @kaushikb11 @rohitgr7

### CODE REVIEW

File: tests/accelerators/test_tpu.py [hunk 1]
```diff
@@ -313,7 +313,6 @@ def test_warning_if_tpus_not_used():
         Trainer()
 
 
-@pytest.mark.skip(reason="TODO(@kaushikb11): Optimize TPU tests to avoid timeouts")
```
Comment (DuYicong515) [inline tests/accelerators/test_tpu.py:316]: is it okay to add "@pl_multi_process_test" here since timeouts are 20min now?

**Review by DuYicong515 — APPROVED** [review]
Thank you!

**Review by carmocca — APPROVED** [review]

**Review by rohitgr7 — APPROVED** [review]

**Review by akihironitta — APPROVED** [review]
